app/views/service_providers/_form.html.slim

```diff
@@ -1,8 +1,21 @@
+- edit_form ||= false
+
 = form.error_notification
 
 - if current_user.admin?
   = form.input :approved, as: :radio_buttons
-  = form.association :user_group
+
+- if edit_form
+  = form.association :user_group,
+    as: :collection_select,
+    collection: current_user.scoped_user_groups,
+    include_blank: ''
+-else
+  = form.association :user_group,
+    as: :collection_select,
+    collection: current_user.scoped_user_groups,
+    include_blank: '',
+    selected: current_user.user_group_id
```

Comment:
I think you can avoid this whole if / else by setting a default value of `current_user.user_group` like: http://stackoverflow.com/a/19029403